src/KygekDev/Height/Height.php

```diff
@@ -31,18 +31,15 @@ public function onCommand(CommandSender $sender, Command $command, string $label
         if ($command->getName() !== "height" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
 
         if (!isset($args[0])) {
-            if (!$sender instanceof Player) {
-                $this->sendMessage($sender, "Usage: /height <player>");
-                return true;
-            }
+            if (!$sender instanceof Player) return false;
```

Comment:
What will it print if the command returns `false`?

Reply:
print `Usage: /height <player>`

> https://github.com/NhanAZ/Height/blob/main/plugin.yml#L21